JSDemos/Demos/Charts/Area/description.md

```diff
@@ -1 +1,12 @@
-Area series can be&nbsp;used to&nbsp;clearly emphasize change in&nbsp;values. In&nbsp;this example, you can switch between the _area_, _stacked area_ and _full-stacked area_ series types. _Stacked area_ series help visualize multiple series and compare the manner in&nbsp;which each series contributes to&nbsp;the total aggregate value for specific arguments. _Full-stacked area_ series help compare the percentage value of&nbsp;multiple area series for each argument.
\ No newline at end of file
+The Chart component can display various [series types](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type). This demo shows the following series types (use a drop-down editor below the Chart to select the type):
```

Comment:
The first sentence seemed redundant to me - we can go on directly to the second, and still include the link. I don't think you want to mention the fact that "chart can display various series types" in the majority of the demos. 

```suggestion
This demo shows different Area [series types](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type). Use a drop-down editor below the Chart to select the type.
```